types/ej.web.all/index.d.ts

```diff
@@ -11692,6 +11702,26 @@ declare namespace ej {
          */
         gotoPage(pageIndex: number): void;
 
+        /** goToFirstPage() helps to navigate to the first page of the pager.
+         * @returns {void}
```

Comment:
JSDoc types are ignored in typescript files, so it's a bad idea to include them since they can easily be wrong. (The rest of the file contains lots of JSDoc, so this is just a note for future reference.)